webapp/locales/pt.json

```diff
@@ -321,7 +321,63 @@
             "disabledBy": "desativado por",
             "reasonCategory": "Categoria",
             "reasonDescription": "Descrição",
-            "createdAt": "Data"
+            "status": "",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-17T15:03:47Z" title="Friday, January 17th 2020, 4:03:47 pm +01:00">Jan 17, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -321,7 +321,63 @@
             "disabledBy": "desativado por",
             "reasonCategory": "Categoria",
             "reasonDescription": "Descrição",
-            "createdAt": "Data"
+            "status": "",
```


@Mogge we must make sure that we don't add empty strings to our locales. Either leave it out or add `"status": null` to the translation files. Otherwise we disable the English fallback and the user will see a blank page.

Reply:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-01-17T17:43:01Z" title="Friday, January 17th 2020, 6:43:01 pm +01:00">Jan 17, 2020</time>_

---


I check this. I tried to reconfigure the @appinteractive , just to match the new semantic requirements but failed so far